src/Menu/select-menu.md

```diff
@@ -24,14 +24,14 @@ The ``Modal`` brings focus to the first menu element upon the click of the trigg
   <MenuItem iconBefore={Add}>A Menu Item With an Icon Before</MenuItem>
   <MenuItem iconAfter={Check}>A Menu Item With an Icon After </MenuItem>
   <MenuItem disabled>A Disabled Menu Item</MenuItem>
-  <MenuItem as={Hyperlink} href="https://en.wikipedia.org/wiki/Hyperlink">A Link Menu Item</MenuItem>
+  <MenuItem as="a" href="https://en.wikipedia.org/wiki/Hyperlink">A Link Menu Item</MenuItem>
 </SelectMenu>
 ```
 
 #### Linked variant
 
 ```jsx live
-<SelectMenu isLink={true} defaultMessage="Choose Your New Best Friend">
```

Comment:
why did you remove this? `isLink` is a valid prop I think (you can probably remove `={true}` part to improve code-style, but not the whole prop)

Reply:
Ok, thanks